static/app/views/dashboardsV2/widgetCard/widgetQueries.tsx

```diff
@@ -427,11 +428,14 @@ class WidgetQueries extends React.Component<Props, State> {
     const {loading, timeseriesResults, tableResults, errorMessage} = this.state;
 
     const filteredTimeseriesResults = timeseriesResults?.filter(result => !!result);
-    return children({
-      loading,
-      timeseriesResults: filteredTimeseriesResults,
-      tableResults,
-      errorMessage,
+    return getDynamicText({
+      value: children({
+        loading,
+        timeseriesResults: filteredTimeseriesResults,
+        tableResults,
+        errorMessage,
+      }),
+      fixed: <div />,
```

Comment:
Hm, this is causing acceptance test failures because the drag handle is on the widget's `ToolbarPanel` after it gets loaded. We could get around the loading issue by forcing a state with the `fixed` variable:
```js
      fixed: children({
        loading: false,
        tableResults: [],
        errorMessage,
        timeseriesResults: [],
      }),
```

Reply:
I actually think we can get away with not returning fixed here (I'd like to return actual data for big number widget resize tests)